static/templates/subscription_settings.handlebars

```diff
@@ -16,9 +16,9 @@
                 <span class="checkmark" data-finish-editing=".stream-name-editable">✓</span>
                 {{/if}}
             </div>
-            <button class="button small rounded subscribe-button sub_unsub_button {{#unless subscribed }}unsubscribed{{/unless}}" type="button" name="button">
+            <button class="button small rounded subscribe-button sub_unsub_button {{#unless subscribed }}unsubscribed{{/unless}}" type="button" name="button" title="{{t 'Subscribe to/unsubscribe from stream'}} (S)">
```

Comment:
I think we probably want to have it just describe the hotkey for the current action (i.e. add another `#{{unless subscribed}}` rather than documenting the hotkey as doing both..